index.js

```diff
@@ -6,23 +6,131 @@
 
 var program = require('commander');
 var colors = require('colors');
+var OnboardingCli = require("./onboardingCli");
+var TranslatorCli = require("./translatorCli");
+var fs = require('fs');
+var helpers = require('./helpers');
+var translatorCli = new TranslatorCli();
```

Comment:
By design, TranslatorCli is the class, translatorCli is the instance. It's a pattern we have been using.  I'm happy to use a different pattern, please recommend one!

---
In reply to: [78631863](https://github.com/openT2T/opent2t-cli/pull/9#discussion_r78631863) [](ancestors = 78631863)